src/main/java/org/openstreetmap/atlas/utilities/conversion/ProtoTagListConverter.java

```diff
@@ -0,0 +1,89 @@
+package org.openstreetmap.atlas.utilities.conversion;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.proto.ProtoTag;
+import org.openstreetmap.atlas.utilities.collections.Maps;
+import org.openstreetmap.atlas.utilities.collections.StringList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Converts back and forth between a List of ProtoTags and an OSM tag Map.
+ *
+ * @author lcram
+ */
+public class ProtoTagListConverter implements TwoWayConverter<List<ProtoTag>, Map<String, String>>
```

Comment:
I dig all the converters

Reply:
I think I doubled the number of converters we have hahaha

Reply:
See `org.openstreetmap.atlas.geography.converters` and `org.openstreetmap.atlas.geography.converters.jts` for some converter love :)